tempesta_fw/tls.c

```diff
@@ -412,9 +404,14 @@ tfw_tls_do_cleanup(void)
 static int
 tfw_tls_start(void)
 {
-	int r;
+	int r = tfw_runstate_is_reconfig();
 
-	if (tfw_runstate_is_reconfig())
+	if (!tfw_tls.crt.version) {
```

Comment:
`tls_certificate` and `tls_certificate_key` directives don't support live reconfiguration, why check this condition on every `tfw_tls_start()` call? Most times it's called during reconfig.